src/Controls/src/Core/BindableObject.cs

```diff
@@ -11,18 +11,24 @@
 
 namespace Microsoft.Maui.Controls
 {
-	/// <include file="../../docs/Microsoft.Maui.Controls/BindableObject.xml" path="Type[@FullName='Microsoft.Maui.Controls.BindableObject']/Docs/*" />
+	/// <summary>
+	/// Provides a mechanism by which application developers can propagate changes that are made to data in one object to another, by enabling validation, type coercion, and an event system.
```

Comment:
I think we could make this slightly more concise

```suggestion
	/// Provides a mechanism to propagate data changes from one object to another. Enables validation, type coercion, and an event system.
```